smart-contract/program/src/utils.rs

```diff
@@ -125,6 +128,11 @@ pub fn assert_uninitialized(account: &AccountInfo) -> ProgramResult {
 }
 
 pub fn assert_authorized_seller(seller: &AccountInfo, seller_index: usize) -> ProgramResult {
+    // print all authorized sellers
+    for i in 0..AUTHORIZED_BOND_SELLERS.len() {
```

Comment:
`for (i, <item>) in AUTHORIZED_BOND_SELLERS.iter().enumerate()` is a bit more idiomatic (cf `cargo clippy`)

Reply:
Cool stuff, thanks for the tip! Anyways, I have deleted this as it was used only for debugging purposes